File: 7-kyu/80's-kids-#2--help-alf-find-his-spaceship/javascript/solution.js
const findSpaceship = map => {
  if (map) {
    const mapSplit = map.split("\n");
    for (let i = 0; i < mapSplit.length; i++) {
      const xCoord = mapSplit[i].split("").findIndex(val => val === "X");
      if (xCoord !== -1) {
        const yCoord = mapSplit.length - i - 1;
        return [xCoord, yCoord];
      }
    }
  }
  return "Spaceship lost forever.";
};

File: 7-kyu/80's-kids-#2--help-alf-find-his-spaceship/python/solution.py
def find_spaceship(astromap):
    if not astromap:
        return 'Spaceship lost forever.'
    astromap_split = astromap.split('\n')
    for idx, row in enumerate(astromap_split):
        if 'X' in row:
            x = row.index('X')
            y = len(astromap_split) - idx - 1
            return [x, y]
    return 'Spaceship lost forever.'
